packages/sources/svelte/package.json

```
@@ -65,6 +65,7 @@
     "./actions/VtmnButton/VtmnButton.svelte": "./dist/VtmnButton.js",
     "./actions/VtmnLink/VtmnLink.svelte": "./dist/VtmnLink.js",
     "./forms/VtmnTextInput/VtmnTextInput.svelte": "./dist/VtmnTextInput.js",
-    "./overlays/VtmnPopover/VtmnPopover.svelte": "./dist/VtmnPopover.js"
+    "./overlays/VtmnPopover/VtmnPopover.svelte": "./dist/VtmnPopover.js",
+    "./indicators/VtmnPrice/VtmnPrice.svelte": "./dist/VtmnPrice.js"
```

For alphabetical order :)
```suggestion
    "./indicators/VtmnPrice/VtmnPrice.svelte": "./dist/VtmnPrice.js",
    "./overlays/VtmnPopover/VtmnPopover.svelte": "./dist/VtmnPopover.js"
```